src/ayab/com.cpp

```diff
@@ -297,7 +297,10 @@ void Com::h_cnfLine(const uint8_t *buffer, size_t size) {
 
   if (GlobalKnitter::setNextLine(lineNumber)) {
     // Line was accepted
-    bool flagLastLine = bitRead(flags, 0U);
+
+    // A Python bool is represented as a byte 
+    // so read the second bit to get the value
+    bool flagLastLine = bitRead(flags, 1U);
```

Comment:
Reading bit 1 and assuming the lastLine flag there is violating the [API](https://github.com/AllYarnsAreBeautiful/ayab-manual/blob/c03c9f76e045b70c3f2d8697ec844a5243b8aaa8/docs/development/API.md?plain=1#L98).

Assuming the error is really caused by a faulty Bool->int conversion in the Python (ayab-desktop) part of AYAB, it should be fixed there to comply with the API (likely https://github.com/AllYarnsAreBeautiful/ayab-desktop/blob/5e781f109c8c86f6190152cfa6db0095c888b242/src/main/python/main/ayab/engine/control.py#L212).